redaxo/src/core/lib/be/controller.php

```diff
@@ -442,8 +462,6 @@ public static function includeCurrentPage()
 
     /**
      * Includes the sub-path of current page.
-     *
-     * @return mixed
```

Comment:
die beiden return-mixed-phpdocs in der datei haben ihre berechtigung.
du hast die in beiden PRs entfernt. Macht das der cs-fixer bei dir? sollte er eigentlich nicht. hat er nur kurz mal gemacht in irgendeiner version, meine ich. aber habe dann unsere config dafür angepasst.

Reply:
Inzwischen auch im dritten PR ;)